operations/setup.go

```diff
@@ -41,15 +42,17 @@ func (c *serviceConf) export() *cedar.Configuration {
 func (c *serviceConf) getSenders(ctx context.Context, conf *model.CedarConfig) (send.Sender, error) {
 	senders := []send.Sender{}
 
-	if c.interactive {
-		senders = append(senders, send.MakeNative())
-	} else {
-		sender, err := send.MakeDefaultSystem()
-		if err != nil {
-			return nil, errors.WithStack(err)
-		}
+	if !c.disableLocalLogging {
+		if c.interactive {
```

Comment:
I think we can ignore local logging when in interactive mode—this is used for the interactive CLI commands run manually (like loading/dumping the cedar DB config to/from a file). In which case, sending logs to stdout is useful always.

```
if c.interactive {
        ...
} else if !c.disableLocalLogging {
        ...
}
```

Reply:
Ah gotcha. Just separated these out.